src/core/redux/selectors/selectOrgsDataSets.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * @flow
+ */
+
+import { List, Map, getIn } from 'immutable';
+import type { UUID } from 'lattice';
+
+import { EDM, ORG_DATA_SETS } from '../constants';
+
+export default function selectOrgsDataSets(dataSetsByOrgId :Map<UUID, List<UUID>>) {
```

Comment:
We should explore using reselect for these selectors that always return a new reference to something. not a blocker for this pr unless you want to give it a go. we've been allowing some fishy hash hack when these are used in react components